docs/GE_Checklist.md

```diff
@@ -6,40 +6,138 @@ to be taken in order to publish a new FIWARE GE.
 Any doubt or request to the FIWARE Foundation shall be made by sending an email
 to [fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org)
 
--   New entry in the [FIWARE Catalogue](https://catalogue-server.fiware.org)
-    under the corresponding Chapter agreed at TSC Level. See also
+### New Generic Enablers
+
+The following mimimal requirements must be met before a new contribution can be
+considered as an Incubated Generic Enabler.
+
```

Comment:
a Generic Enabler must show how it integrates within the overall FIWARE Architecture

Reply:
Fixed 7228c75 - Additional bullet added